edge-hub/docker/linux/amd64/Dockerfile

```diff
@@ -10,7 +10,8 @@ RUN apk update && \
     apk add --no-cache snappy libcap
 
 # Add an unprivileged user account for running Edge Hub
-RUN adduser -Ds /bin/sh edgehubuser 
+ARG EDGEHUBUSER_ID=1000
```

Comment:
Does this actually help? The container user namespace will likely always have 1000 available. I believe the issue is not about the container user, but the mapped uid on the host.

Reply:
As discussed, seems like Docker *does* have a one-to-one mapping between host and container uids.